app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -99,17 +105,16 @@ public static void open(@NonNull final Context context) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(layout.activity_search);
+        ButterKnife.bind(this);
         Option.ofObj(savedInstanceState)
               .ifNone(this::addSearchFragment);
 
-        Toolbar toolbar = (Toolbar) findViewById(id.toolbar_search);
+        Toolbar toolbar = ButterKnife.findById(this, id.toolbar_search);
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
-        searchView = get((SearchView) findViewById(id.search_view));
-        searchView.setIconified(false);
-        closeButton = (ImageView) searchView
-                .findViewById(android.support.v7.appcompat.R.id.search_close_btn);
+        get(searchView).setIconified(false);
```

Comment:
Is the get necessary? Won't the binding fail with an Exception if it doesn't exist? Sure the field is technically nullable because we don't assign as final constructor, but it would take some asynchronous gymnastics for it to ever occur. I think that we are better off not using that check for ButterKnife bound views unless there's an explicit way path for them to be null.